doc/whats_new/v0.20.rst

```diff
@@ -15,6 +15,12 @@ enhancements to features released in 0.20.0.
 Changelog
 ---------
 
+:mod:`sklearn.cluster`
+......................
+
+- |Fix| Fixed a bug in :class:`cluster.KMeans` to use available threads.
```

Comment:
please mention: when ``n_jobs > 1``

Reply:
And maybe we need to explain the bug: ``Fixed a bug where XXX is not parallelized when n_jobs > 1``.

Reply:
Sure, I think `n_jobs = -1` will also try to use all the threads

Reply:
Hope this is fine
```
- |Fix| Fixed a bug in :class:`cluster.KMeans` where computation was single
  threaded when `n_jobs > 1` or `n_jobs = -1`.
```